app/views/projects/history/_project_history_modal.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= viral_dialog(open: true, size: :large) do |dialog| %>
+  <%= dialog.with_header(
+    title: t(".version_number", version_number: log_data[:version])
+  ) %>
```

Comment:
**nitpick:** The alignment seems to be off on the modal, I checked lookbook and it seems like it should be all lined up, can you take a look and see what is causing it:
![image](https://github.com/phac-nml/irida-next/assets/11295750/9478e822-bffd-4790-8c88-c7b82e8ded22)

Reply:
Fixed in 1e5a9b1dd9dd0238ef206c505556c1accb550c51. Different padding values were set for the header, section, and footer in the tailwind css file